### ISSUE TITLE
PYODBC not connecting from Linux to query Windows SQL Server.

### ISSUE THREAD

**Issue by MaximoTrinidad** [mkleehammer/pyodbc#162]
Just to be clear! During a presentation I ended up trying to run a query from Linux Python to a Windows SQL Server which error-out.

I got PYODBC installed in both Linux and Windows. It does work Windows to Windows SQL Server, and Linux to Linux SQL Server. But failed with a connection timeout when trying Linux to Windows SQL Server.

So, my question is: Can it be done? Or, its possible my Linux system got somehow misconfigured.

:)

**Comment by mkleehammer** [comment]
It absolutely can be done.  A connection timeout sounds more like simple firewall or port issues. 

In fact, I don't think new SQL Server installs enable TCP/IP by default, but that could have changed over the years .  (Local connections would be made with named pipes.).

**Comment by MaximoTrinidad** [comment]
Here's a sample image:

![pyodbclinuxnotwindows](https://cloud.githubusercontent.com/assets/19892435/20984524/a1b09ba2-bc8e-11e6-865d-0440fd85863f.png)

Interesting, I make sure I got everything enabled (Pipe / TCP/IP) and I'm using SQL Authentication. But I reallized that I have one difference. My laptop is joined to a domain while the other is a Workgroup. 

Sorry! I got caught in the moment. :P
